app/views/index.njk

```diff
@@ -1,89 +1,98 @@
+{% set pageTitle = "Title needed" %}
+{% set pageDescription = "This is the lead description needed" %}
+
 {% extends 'includes/layout.njk' %}
 
 {% block outerBody %}
-<section class="nhsuk-hero">
-  <div class="nhsuk-width-container nhsuk-hero--border">
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
-        <div class="nhsuk-hero__wrapper">
-          <h1 class="nhsuk-u-margin-bottom-3">Technical Review Group and Governance</h1>
-          <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">Helping technical architects and project professionals to navigate our technical governance process.</p>
-        </div>
-      </div>
-    </div>
+  <nav class="nhsuk-breadcrumb{% if params.classes %} {{ params.classes }}{% endif %}" aria-label="Breadcrumb"
+  {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+  <div class="nhsuk-width-container">
+    <ol class="nhsuk-breadcrumb__list">
+      {%- for item in params.items %}
+        {%- if item.href %}
+          <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="/"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>Home</a>{% if not loop.last %}{% endif %}</li>
+        {%- endif -%}
+      {% endfor %}
+      <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="/"{% for attribute, value in item.attributes %} {{attribute}}="{{value}}"{% endfor %}>Home</a></li>
+      </ol>
   </div>
-</section>
+  </nav>
 {% endblock %}
 
+
 {% block body %}
 <div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-three-quarters">
-<h2 id="how-this-manual-could-help-you">How this manual could help you</h2>
-    <p>As a Technical Architect or Project lead:</p>
+  <div class="nhsuk-grid-column-full">
```

Comment:
We should use `nhsuk-grid-column-three-quarters` here so that the content doesn't exceed around 70 to 80 characters so that it's easier to read.